src/test/java/com/practise/tdd/WordCountShould.java

```diff
@@ -13,7 +14,7 @@ public void setup() {
 	}
 	
 	@Test
-	public void count_unique_words_in_single_word_string() {
-		
+	public void return_unique_words_in_single_word_string() {
```

Comment:
But the test case name is different from the asserts. Why do you need 1 appended here?